src/System.Text.Json/ref/System.Text.Json.csproj

```diff
@@ -5,7 +5,7 @@
   <ItemGroup>
     <Compile Include="System.Text.Json.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' != 'netstandard' AND '$(TargetsNetFx)' != 'true'">
```

Comment:
Is this change necessary now?

It is better for the condition to clearly be a negative of the other (like if/else).
```csproj
<ItemGroup Condition="...NOT X...">
</ItemGroup>

<ItemGroup Condition="...X...">
</ItemGroup>
```